docs/rest_api.rst

```diff
@@ -183,19 +182,20 @@ Querying Information About Channels and Tokens
       HTTP/1.1 200 OK
       Content-Type: application/json
 
-      [
-          {
-              "channel_address": "0x2a65Aca4D5fC5B5C859090a6c34d164135398226",
-              "partner_address": "0x61C808D82A3Ac53231750daDc13c777b59310bD9",
-              "token_address": "0xEA674fdDe714fd979de3EdF0F56AA9716B898ec8",
-              "balance": 35000000,
-              "state": "open",
-              "settle_timeout": 100,
-              "reveal_timeout": 30
-          }
-      ]
+      {
+          "channel_address": "0x2a65Aca4D5fC5B5C859090a6c34d164135398226",
+          "partner_address": "0x61C808D82A3Ac53231750daDc13c777b59310bD9",
+          "token_address": "0xEA674fdDe714fd979de3EdF0F56AA9716B898ec8",
+          "balance": 35000000,
+          "state": "open",
+          "settle_timeout": 100,
+          "reveal_timeout": 30
+      }
 
    :statuscode 200: Successful query
+   :statuscode 404:
+    - Given channel address is not a valid eip55-encoded Ethereum address or
```

Comment:
Change this: It's not the channel address but any of the provided addresses. i.e.: partner and/or token